### DISCUSSION TITLE
problem while using sudo so-learn list command

### DISCUSSION THREAD

**Discussion by lembratour** [Security-Onion-Solutions/securityonion#12913] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

16

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

hi guys , I'm encountering an issue with the sudo so-learn command not being found when I attempt to run it on my Security Onion system. This command is crucial for me to perform certain tasks related to machine learning and threat detection 
https://docs.securityonion.net/en/2.3/machine-learning.html

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
Security Onion 2.4 does not include so-learn. The docs you linked to are for Security Onion 2.3, which is EOL.

If you create a discussion describing your end goal and maybe steps you've taken on your own. Others in the community maybe able to guide you in the right direction
